index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
+      <div class="input-username">
+        <input
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
+      placeholder="Full name"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      <div class="input-username">
+        <input 
+      class="input-1"
+      type="email"
+      placeholder="Email address"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      
+      <textarea class="textarea" maxlength="500">Enter text here</textarea>
```

Comment:
- [ ]  Kindly 😊 try to remove the content of the textarea because you make it static and the user has to remove it to write his info, so please try to make it placeholder like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My form | Your form| 
| --- | --- |
|  ![Recording 2023-02-28 at 22 11 10](https://user-images.githubusercontent.com/83360636/221967518-c31167ea-7cc7-4858-bdae-c7c61f788b05.gif)  |    ![Recording 2023-02-28 at 22 11 57](https://user-images.githubusercontent.com/83360636/221967688-47d10c58-6f44-4419-88e4-5c1ea529875f.gif)  |